### ISSUE TITLE
Password

### ISSUE THREAD

**Issue by hez2010** [SYSU-MSC-Studio/FreshBoard#2]
Can we set password to 123456?

**Comment by SgLy** [comment]
I think it is too simple. What about `456123` or something like this?

**Comment by hez2010** [comment]
@SgLy Nope. After twice consideration, I think we should add some alphabet characters.
I think 4a5b1c2d is okay.
